src/ethereum-providers/connectors.ts

```diff
@@ -6,7 +6,20 @@ const RINKEBY_ETH_NODE = env('RINKEBY_ETH_NODE')
 const XDAI_ETH_NODE = env('XDAI_ETH_NODE')
 const POLYGON_ETH_NODE = env('POLYGON_ETH_NODE')
 
-export const CONNECTORS = [
+type ConnectorProviderType = {
+  [key: string]: any
+  id: string
+  properties: {
+    [key: string]: any
```

Comment:
Strong types already knowed but with dynamic to ensure it now will break anything